src/Compilers/CSharp/Portable/Symbols/Synthesized/SynthesizedAccessorValueParameterSymbol.cs

```diff
@@ -24,6 +25,26 @@ public SynthesizedAccessorValueParameterSymbol(SourceMemberMethodSymbol accessor
         {
         }
 
+        internal override FlowAnalysisAnnotations FlowAnalysisAnnotations
+        {
+            get
+            {
+                var result = FlowAnalysisAnnotations.None;
```

Comment:
>var result = FlowAnalysisAnnotations.None; [](start = 16, length = 42)

Implementation of this property should rely on well known attributes infrastructure.